tests/kernel/ib_tests.pm

```diff
@@ -105,7 +105,7 @@ sub run {
     $self->select_serial_terminal;
 
     # unload firewall. MPI- and libfabric-tests require too many open ports
-    script_run("systemctl stop firewalld");
+    systemctl("stop " . opensusebasetest::firewall);
```

Comment:
systemctl function should be imported by line 18:

    use utils;

Reply:
Looks good to me